src/app/Shared/Model/Api_Filter.model.ts

```diff
@@ -3,7 +3,7 @@ export interface FilterConfigDTO {
         id: string;
         name: string;
         parent: string;
-        target: string;
+        trigger: string;
```

Comment:
I would not say this is the better name...

Reply:
Dominic and I think this is better because it triggers the corresponding subfilter.

Reply:
I think we had this discussion sometimes ago and i dismissed trigger as a name and proposed target. Trigger is kind of reversed for event like entities. So e.g. an error message banner could be triggered by an api failure. But it is more often used to describe triggering a chain of state transitions.
So the main problem with the name i have is, that it has behavioral character and gives the intention, as you stated, that the filterConfig can be triggered. But it is a simple data object holding data for describing the filter and should not wire in behavior. So what if the front-end does show all possible filters (as in the beginning), then the name would make no sense.
As also stated sometimes ago a more simpler approach would be to inverse the logic:
```
FilterDefinition: {
  value: string,
  subFilter?: Filter
}
Filter: {
  name: string,
  definitions: FilterDefinition[]
}  
```
(FilterDefinition is a bad name too :) ) So no parent, target or trigger.
This comment is just for give you some thinking :) As for the API is private and i think you all know what is meant by trigger the name might be fine.

Reply:
But the front end only shows the subfilter selector if the "trigger" was previously selected in a MainFilter selector. So it actually changes the behavior of the frontend.

Reply:
Yes, actually it does. I thought of another front-end or changing the front-end so it always shows all filters. Then it would make no sense. So, the API should be independent of the front-end implementation details and a REST API should only describe data and not behavior